dh-keys.js

```diff
@@ -45,4 +73,35 @@ module.exports = class DHKeys {
 
     return { secret: _sk, public: _pk }
   }
+
+  toTFK () {
+    console.log('toTFK deprecated, please use toBFE')
+    return this.toBFE()
+  }
+}
+
+function bufferize (key, length) {
+  if (Buffer.isBuffer(key)) return key
+
+  if (typeof (key) !== 'string') throw new Error(`unable to bufferize ${typeof key}`)
+
+  return Buffer.from(
+    key
+      .replace('.ed25519', '')
+      .replace('.bbfeed-v1', ''),
```

Comment:
same as the other commit, we don't need the bbfeed check here

Reply:
done